### PR TITLE
Added two user stories

### PR DESCRIPTION
**Pull request by mspatelibm**
First user story captured if Jessica received a new device (5.2.15)

Second user story captured if Jessica wanted to delete/deregister a device (5.2.16)

### CODE REVIEW

File: hld.md [hunk 1]
```diff
@@ -421,6 +421,14 @@ _coming soon_
 
 _coming soon_
 
+#### 5.2.15 As a Keycloak end user (Jessica), I want to be able to modify my device registration and MFA enrollment in the case of getting another device, phone number, and/or email (IBM Verify)
+
+_coming soon_
+
+#### 5.2.16 As a Keycloak end user (Jessica), I want to be able to delete my devices/disassociate email and phone number after enrolling to the various authenticators (IBM Verify).
```
Comment (afbruch-ibm) [inline hld.md]: These both aren't 100% scoped to IBM Verify, right? They apply to Verify, FIDO, Email, and SMS? If that's correct, I think we should either remove `(IBM Verify)` from the suffix of each scenario, _or_ explicitly call out each backing technology that's affected (Verify, FIDO, Email, SMS, etc.)

**Comment by mspatelibm** [comment]
@afbruch-ibm @jason-choi1 added details per comments above - please let me know if you agree and can commit the changes to master

File: hld.md [hunk 2]
```diff
@@ -421,14 +421,20 @@ _coming soon_
...
+#### 5.2.15 As a Keycloak end user (Jessica), I want to be able to modify my device registration and MFA enrollment in the case of getting another device for IBM Verify Registration (QRCode and Mobile Push)
 
 _coming soon_
 
-#### 5.2.16 As a Keycloak end user (Jessica), I want to be able to delete my devices/disassociate email and phone number after enrolling to the various authenticators (IBM Verify). 
+#### 5.2.16 As a Keycloak end user (Jessica), I want to be able to modify my device registration and MFA enrollment in the case of getting another device for FIDO registation
 
 _coming soon_
 
+#### 5.2.17 As a Keycloak end user (Jessica), I want to be able to modify my email for TOTP in case of wanting to change where my TOTP code gets delivered to
```
Comment (afbruch-ibm) [inline hld.md]: I suspect you mean OTP, not TOTP? TOTP is a code generated _on_ a device like Google Authenticator or IBM Verify. TOTP's are not delivered to an email address or a phone number. Similar below for 5.2.18.

**Comment by mspatelibm** [comment]
@afbruch-ibm modified to OTP

**Pull request merged**